### ISSUE TITLE
CRAN release

### ISSUE THREAD

**Issue by seabbs** [brittanyblouin/ANCRTAdjust#21]
This is not a blocker for JOSS publication but are there any plans for a CRAN release of this software? 

In general being on CRAN is an important marker of quality (at least basic quality) and makes installation much easier. 

For this # [openjournals/joss-reviews/issues/1740](https://github.com/openjournals/joss-reviews/issues/1740)  review

**Comment by m-maheu-giroux** [comment]
In the medium term, yes, we intend to submit to CRAN. However, we would first like to obtain users feedback on functions before doing so. Publishing the package on JOSS and github first seems like the appropriate sequence.

**Comment by seabbs** [comment]
Great - good to hear. 

Have you checked that everything is passing CRAN checks? (I use `devtools::check_rhub()` for this)

**Comment by ellessenne** [comment]
+1 for checking on R-Hub - you may get some weird errors/notes when testing on different platforms.
On a side note, I would actually suggest using `rhub::check_for_cran()` instead of the function from the `devtools` package (I think many functions from `devtools` are being relocated to other packages? Not sure if this is one of those).

**Comment by seabbs** [comment]
That is probably a good idea. I think you are correct but I am not sure what is moving and what isn't. Just been plugging along as normal until something stops working ...

**Comment by ellessenne** [comment]
Good point! I believe everything still works for now, so either will do 😃
